ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -298,6 +298,13 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-edge-zones-byo-vpc
   timeout: 6h0m0s
+- always_run: true
```

Comment:
do you really want this to always run?

Reply:
Yes when AWS (run_if_changed: aws).
Those jobs are not provisioning infra, only consuming AWS API requests to evaluate many scenarios that can broke Local or Wavelength zones clusters, specially in untested regions by CI but declared as supported in OCP.